share/zm-backend.sh-mysql

```diff
@@ -14,6 +14,13 @@
 #                    database engine.
 ### END INIT INFO
 
+# Unset potentially conflicting locale, setting the default
+unset LANGUAGE
+unset LANG
+unset LC_MESSAGES
+unset LC_ALL
+export LC_CTYPE="en_US.UTF-8"
+
```

Comment:
This shouldn't be done in a wrapper script but inside the daemon itself.

Is there a reason why you set LC_CTYPE to en_US.UTF-8?

Reply:
I agree with you that it should be done in the JSON RPC daemon itself, but that is too large change to do now. This is a work-around that makes it work.

en_US.UTF-8 is the default locale selected by Backend unless some other language is selected. I have a previous experience that you get problem with Backend unless it starts in a UTF-8 environment.

Reply:
@mattias-p, what changes do you request?

Reply:
In that case I request that comments be added about the fact that it's a workaround, and previous problems with non-UTF-8 environments.